lib/datadog/tracing/sampling/rule_sampler.rb

```diff
@@ -30,15 +30,20 @@ def initialize(
           default_sampler: nil
         )
           @rules = rules
-          @rate_limiter = if rate_limiter
+          @rate_limiter = if Datadog.configuration.appsec.standalone.enabled
+                            # 0.0167 ~ 1 trace per minute
+                            Core::TokenBucket.new(0.0167, 1.0)
```

Comment:
for readability I would propose to use `1/60` instead of `0.0167` here

Reply:
Fixed!